### PR TITLE
Integrate all recent comments

### PR DESCRIPTION
**Pull request by gvanas**
I tried to address the pending comments on the KangarooTwelve and TurboSHAKE draft. More precisely, I considered:
- John Mattsson's emails of October 24 and 31 on CFRG, except for the definition of a new 256-bit variant of K12;
- David Benjamin's email of November 15 on CFRG;
- Nick Sullivan's review comments of November 14;
- David Wong's updated affiliation;
- Joan Daemen's editorial comments.

### CODE REVIEW

File: draft-irtf-cfrg-kangarootwelve.xml [hunk 1]
```diff
@@ -342,13 +342,13 @@
         <t>The Customization string MAY serve as domain separation.
         It is typically a short string such as a name or an identifier (e.g. URI,
         ODI...).
-        It can serve the same purpose as TurboSHAKE's D input (see <xref target="TurboSHAKE_Interface"/>), but with a larger range.
+        It can serve the same purpose as TurboSHAKE's D input parameters (see <xref target="TurboSHAKE_Interface"/>), but with a larger range.
```
Comment (gvanas) [inline draft-irtf-cfrg-kangarootwelve.xml]: parameters → parameter

File: draft-irtf-cfrg-kangarootwelve.xml [hunk 2]
```diff
@@ -342,13 +342,13 @@
...
         </t>
 
         <t>By default, the Customization string is the empty string. For an API that
         does not support a customization string parameter, C MUST be the empty string.</t>
 
-        <t>Note that an implementation MAY propose an interface with input and/or output incrementality as specified in <xref target="TurboSHAKE_Interface"/>.</t>
+        <t>Note that an implementation MAY propose an interface with input parameters and/or output incrementality as specified in <xref target="TurboSHAKE_Interface"/>.</t>
```
Comment (gvanas) [inline draft-irtf-cfrg-kangarootwelve.xml]: Here "with input and/or output incrementality" does not require "parameters".

Comment (ildyria) [reply draft-irtf-cfrg-kangarootwelve.xml]: Fixed: [ce6bc86](https://github.com/cfrg/draft-irtf-cfrg-kangarootwelve/pull/9/commits/ce6bc861f3d431b1823a1abee1de1f07e0c643e7)

File: draft-irtf-cfrg-kangarootwelve.xml (hunk 1 above)
Comment (ildyria) [reply draft-irtf-cfrg-kangarootwelve.xml]: Fixed: [ce6bc86](https://github.com/cfrg/draft-irtf-cfrg-kangarootwelve/pull/9/commits/ce6bc861f3d431b1823a1abee1de1f07e0c643e7)

**Comment by gvanas** [comment]
Fine with me now, thanks!

**Pull request merged**